content/docs/start/data-versioning.md

```diff
@@ -7,7 +13,16 @@ with the same performance as with small code files? Imagine you can do a
 than a second?
 
 The core part of DVC is a few commands that you can run along with Git to track
-a large file, ML model or a directory. Read on, or watch our video docs!
+a large file, ML model, or a directory. Think "Git for data". For example, you
+can use DVC and Git to:
+
+- Track large files and directories in Git with the same performance as small
+  code files.
+- See data files and ML model files next to code in your workspace with
+  `git clone`.
+- Switch between versions of a 100GB dataset instantly with `git checkout`.
+
+Read on or watch our video docs to learn about versioning data with DVC!
```

Comment:
We can definitely insert "Git for data" but everything else here is a huge change. Yeah sorry, Get Started pages need to be as simple as possible, this is too much in this case (and changes the precious tone/meaning as Ivan mentioned). I only wanted to do a very small copy edit to improve the language on the first paragraph (but again, never mind about that, I'll take care of it somewhere else).

Reply:
p.s. It's my bad for expanding the scope...

Reply:
p.p.s. here's basically what I had in mind: https://github.com/iterative/dvc.org/pull/1871/files#diff-ea052227e82952e1865a66baab65e3ef88513639e7a00d7dedab4e44c53e0891 cc @shcheklein

Reply:
OK, PTAL at latest version that combines your latest and adds 'versioning data' to end of p2.